pyairctrl/airctrl.py

```diff
@@ -1,819 +1,389 @@
             if dt != 0:
-                print('[dt]          Timer: {} hours'.format(dt))
-        if 'dtrs' in status:
-            dtrs = status['dtrs']
+                print("[dt]          Timer: {} hours".format(dt))
+        if "dtrs" in status:
+            dtrs = status["dtrs"]
             if dtrs != 0:
-                print('[dtrs]        Timer: {} minutes left'.format(dtrs))
-        if 'fltsts0' in status:
-            fltsts0 = status['fltsts0']
-            print('[fltsts0]     Pre-filter and Wick: clean in {} hours'.format(fltsts0))
-        if 'fltsts1' in status:
-            fltsts1 = status['fltsts1']
-            print('[fltsts1]     HEPA filter: replace in {} hours'.format(fltsts1))
-        if 'fltsts2' in status:
-            fltsts2 = status['fltsts2']
-            print('[fltsts2]     Active carbon filter: replace in {} hours'.format(fltsts2))
-        if 'wicksts' in status:
-            wicksts = status['wicksts']
-            print('[wicksts]     Wick filter: replace in {} hours'.format(wicksts))
-        if 'err' in status:
-            err = status['err']
+                print("[dtrs]        Timer: {} minutes left".format(dtrs))
+        if "fltsts0" in status:
+            fltsts0 = status["fltsts0"]
+            print(
+                "[fltsts0]     Pre-filter and Wick: clean in {} hours".format(fltsts0)
+            )
+        if "fltsts1" in status:
+            fltsts1 = status["fltsts1"]
+            print("[fltsts1]     HEPA filter: replace in {} hours".format(fltsts1))
+        if "fltsts2" in status:
+            fltsts2 = status["fltsts2"]
+            print(
+                "[fltsts2]     Active carbon filter: replace in {} hours".format(
+                    fltsts2
+                )
+            )
+        if "wicksts" in status:
+            wicksts = status["wicksts"]
+            print("[wicksts]     Wick filter: replace in {} hours".format(wicksts))
+        if "err" in status:
+            err = status["err"]
             if err != 0:
-                err_str = {49408: 'no water', 32768: 'water tank open', 49155: 'pre-filter must be cleaned'}
+                err_str = {
+                    49408: "no water",
+                    32768: "water tank open",
+                    49155: "pre-filter must be cleaned",
+                }
                 err = err_str.get(err, err)
-                print('-'*20)
-                print('[ERROR] Message: {}'.format(err))
+                print("-" * 20)
+                print("[ERROR] Message: {}".format(err))
 
     def set_values(self, values, debug=False):
-        if debug:
-            self.coapthon_logger.setLevel("DEBUG")
-        for key in values:
-            self._set(key, values[key])
+        self._client.set_values(values, debug)
 
     def get_status(self, debug=False):
-        if debug:
-            self.coapthon_logger.setLevel("DEBUG")
-        status = self._get()
+        status = self._client.get_values(debug)
```

Comment:
this should be `get_status` instead of `get_values`. Will commit your PR as is and will fix it in a later patch.